spcs-basic/README.md

```diff
@@ -0,0 +1,71 @@
+# Instructions
+
+## Set your SnowCLI connection (optional)
+
+We use your default connection to connect to Snowflake and deploy the images / app. Set your
+default connection:
+
+```sh
+export SNOWFLAKE_DEFAULT_CONNECTION_NAME=na_provider_consumer_qa6
+```
+
+## Create image repository, build and push our local service image
+
+The [service/](service/) directory contains a [Dockerfile](service/Dockerfile) that builds a
+simple Python server that responds to GET health checks, a GET for `/index.html`, as well as
+POSTing to `/echo` in the Snowflake External Function payload format. You can build it and
+push it to an image repository in your count like so:
+
+```sh
+./build-and-push.sh
+```
+
+This command will always use your default SnowCLI connection.
+
+## Deploy the application
+
+Deploy the app package and instance as such:
+
+```sh
+snow app run
+```
+
```

Comment:
Let's add a note here that the application and package have been automatically suffixed by their (local) user name.